src/Tools/Tools.slnf

```diff
@@ -1,22 +1,23 @@
-﻿{
+{
   "solution": {
     "path": "..\\..\\AspNetCore.sln",
-    "projects" : [
-      "src\\Tools\\dotnet-watch\\src\\dotnet-watch.csproj",
-      "src\\Tools\\dotnet-watch\\test\\dotnet-watch.Tests.csproj",
+    "projects": [
+      "src\\Testing\\src\\Microsoft.AspNetCore.Testing.csproj",
```

Comment:
Added this reference only, VS decided to rearrange everything when I used the UI to add this.